### DISCUSSION TITLE
Bug: resource routing replaces 'ta' with 'um'

### DISCUSSION THREAD

**Discussion by netunof** [laravel/framework#37114] [Q&A] [answered]
Hello there devs, today i found out an annoying problem with routing:

Route::resource('dentista', DentistaController::class);

please note the 'ta' at the end of the word

doing a php artisan route:list i get:

GET|HEAD  | dentista                             | dentista.index    | App\Http\Controllers\DentistaController@index     | web
POST           | dentista                            | dentista.store     | App\Http\Controllers\DentistaController@store     | web
GET|HEAD  | dentista/create                 | dentista.create   | App\Http\Controllers\DentistaController@create   | web
GET|HEAD  | dentista/{dentistum}        | dentista.show     | App\Http\Controllers\DentistaController@show     | web
PUT|PATCH | dentista/{dentistum}        | dentista.update | App\Http\Controllers\DentistaController@update  | web
DELETE       | dentista/{dentistum}        | dentista.destroy | App\Http\Controllers\DentistaController@destroy | web
GET|HEAD  | dentista/{dentistum}/edit | dentista.edit       | App\Http\Controllers\DentistaController@edit        | web

notice the replacing of 'ta' with 'um'. That happens with any word that ends in 'ta'

**Comment by carstenjaksch** [comment, marked as answer]
Created a new issue for that. #37120

**Reply by carstenjaksch** [reply]
Just closed and answered. Please refer to the issue @netunof.
